docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+      * [Section 7.1.2.5 - TLS Subordinate CA Certificate Profile](#7125-tls-subordinate-ca-certificate-profile)
+  * [Section 7.1.2.6 - Subscriber (End-Entity) Certificate Profile](#7126-subscriber-server-certificate-profile)
+    * Domain Validated
+    * Individual Validated
+    * Organization Validated
+    * Extended Validation
+  * __**TBD: Infrastructure Certificates**__
+    * [Section 7.1.2.7 - OCSP Responder Certificate Profile](#7127-ocsp-responder-certificate-profile)
+    * __**TBD: Infrastructure Cert**__
+    * __**TBD: Precert Signing CA**__
+    * __**TBD: Interoperability Testing Certs (valid/revoked/expired)?**__
+
+#### 7.1.2.1 Root CA Certificate Profile
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) containing at least 64 bits of output from a CSPRNG. |
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | Encoded value MUST be byte-for-byde identical to the encoded `subject` |
+| \ \ \ \ `validity`             | __**TBD**__ |
+| \ \ \ \ `subject`              | See [Section 7.1.2.8.1](#71281-ca-certificate-naming) |
+| \ \ \ \ `subjectPublicKeyInfo` | See [Section 7.1.3.1](#7131-subjectpublickeyinfo) |
+| \ \ \ \ `issuerUniqueID`       | MUST NOT be present |
+| \ \ \ \ `subjectUniqueID`      | MUST NOT be present |
+| \ \ \ \ `extensions`           | See [Section 7.1.2.1.1](#71211-root-ca-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST be byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.1.1 Root CA Extensions
+
+| __Extension__                     | __Presence__ | __Critical__ | __Description__ |
+| ----                              | -            | -            | ----- |
+| `authorityKeyIdentifier`          | SHOULD       | N            | See [Section 7.1.2.1.2](#71212-authority-key-identifier) |
+| `basicConstraints`                | MUST         | Y            | See [Section 7.1.2.1.3](#71213-basic-constraints) |
+| `keyUsage`                        | MUST         | Y            | See [Section 7.1.2.8.6](#71286-key-usage) |
+| `subjectKeyIdentifier`            | MUST         | N            | See [Section 7.1.2.9.5](#71295-subject-key-identifier) |
+| `extKeyUsage`                     | MUST NOT     | N            | - |
+| `certificatePolicies`             | SHOULD NOT   | N            | See [Section 7.1.2.8 4](#71284-certificate-policies---affiliated-ca) |
+| Signed Certificate Timestamp List | MAY          | N            | See [Section 7.1.2.9.4](#71294-signed-certificate-timestamp-list) |
```

Comment:
In practice, the time sharded logs (and the movement towards **only** time sharded logs) means that CAs are unlikely to use this. However, it's not forbidden, and since the goal is to get to a MUST NOT for any other extension, this hopefully gets us closer?

Reply:
Adding SCT(s) in a Root Certificate sounds quite challenging but I see your point.